common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -208,10 +213,8 @@ def get_html(self):
         # Video caching is disabled for Studio. User_location is always None in Studio.
         # CountryMiddleware disabled for Studio.
         cdn_url = getattr(settings, 'VIDEO_CDN_URL', {}).get(self.system.user_location)
-
         if getattr(self, 'video_speed_optimizations', True) and cdn_url:
-            cdn_info = getattr(settings, 'VIDEO_CDN_INFO', {}).get(self.system.user_location)
-            branding_info = get_branding_info(cdn_info.get('BRANDING'))
+            branding_info = BrandingInfoConfig.get_config().get(self.system.user_location)
```

Comment:
I think it would be better to pass `self.system.user_location` as an argument to `get_config()`.
`BrandingInfoConfig.get_config(self.system.user_location)`.